tests/unit/test_api.py

```diff
@@ -321,6 +361,11 @@ def test_get_ctms_for_api_example(client, example_contact):
             }
         },
     ),
+    (
+        "GET",
+        "/bulk?start=2020-01-22T03%3A24%3A00%2B00%3A00&end=2021-01-29T09%3A26%3A57.511000%2B00%3A00&limit=1&after=OTNkYjgzZDQtNDExOS00ZTBjLWFmODctYTcxMzc4NmZhODFkLDIwMjAtMDEtMjIgMTU6MjQ6MDArMDA6MDA=",
+        None,
```

Comment:
Thanks for adding this!

Reply:
Modified this from `/bulk` to `/updates`